tests/integration/components/explore/side-bar-test.js

```diff
@@ -0,0 +1,10 @@
+import { test } from 'ember-qunit';
+import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('explore-side-bar', 'Integration | Component | explore side bar');
+
+test('it renders', function(assert) {
+  this.render(hbs`{{explore/side-bar}}`);
```

Comment:
Needs to be `{{explore/side-bar i18n=i18n}}` to pass the test. since we're using the `i18n` service in the component.